pkg/services/mysqldb/arm_template.go

```diff
@@ -131,3 +131,165 @@ var armTemplateBytes = []byte(`
+				"defaultValue": "AllowAll"
+			},
+			"firewallStartIpAddress": {
+				"type": "string",
+				"minLength": 1,
+				"maxLength": 15,
+				"defaultValue": "0.0.0.0"
+			},
+			"firewallEndIpAddress": {
+				"type": "string",
+				"minLength": 1,
+				"maxLength": 15,
+				"defaultValue": "0.0.0.0"
+			},
+			"sslEnforcement": {
+				"type": "string",
+				"allowedValues": [ "Enabled", "Disabled" ],
+				"defaultValue": "Enabled"
+			},
+			"tags": {
+				"type": "object"
+			}
+		},
+		"variables": {
+			"DBforMySQLapiVersion": "2017-04-30-preview"
+		},
+		"resources": [
+			{
+				"apiVersion": "[variables('DBforMySQLapiVersion')]",
+				"kind": "",
+				"location": "[parameters('location')]",
+				"name": "[parameters('serverName')]",
+				"properties": {
+					"version": "[parameters('version')]",
+					"administratorLogin": "azureuser",
+					"administratorLoginPassword": "[parameters('administratorLoginPassword')]",
+					"storageMB": "[parameters('skuSizeMB')]",
+					"sslEnforcement": "[parameters('sslEnforcement')]"
+				},
+				"sku": {
+					"name": "[parameters('skuName')]",
+					"tier": "[parameters('skuTier')]",
+					"capacity": "[parameters('skuCapacityDTU')]",
+					"size": "[parameters('skuSizeMB')]"
+				},
+				"type": "Microsoft.DBforMySQL/servers",
+				"tags": "[parameters('tags')]",
+				"resources": [
+					{
+						"type": "firewallrules",
+						"apiVersion": "[variables('DBforMySQLapiVersion')]",
+						"dependsOn": [
+							"[concat('Microsoft.DBforMySQL/servers/', parameters('serverName'))]"
+						],
+						"location": "[parameters('location')]",
+						"name": "[parameters('firewallRuleName')]",
+						"properties": {
+							"startIpAddress": "[parameters('firewallStartIpAddress')]",
+							"endIpAddress": "[parameters('firewallEndIpAddress')]"
+						}
+					}
+				]
+			}
+		],
+		"outputs": {
+			"fullyQualifiedDomainName": {
+				"type": "string",
+				"value": "[reference(parameters('serverName')).fullyQualifiedDomainName]"
+			}
+		}
+	}
+	`)
+
+// DB only Template
+// nolint: lll
+var dbOnlyArmTemplateBytes = []byte(`
+	{
+		"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+		"contentVersion": "1.0.0.0",
+		"parameters": {
+			"version": {
+				"type": "string",
+				"allowedValues": [ "5.7", "5.6" ],
+				"defaultValue": "5.7"
+			},
+			"location": {
+				"type": "string"
+			},
+			"serverName": {
+				"type": "string",
+				"minLength": 2,
+				"maxLength": 63
+			},
+			"databaseName": {
+				"type": "string",
+				"minLength": 2,
+				"maxLength": 63
+			},
+			"tags": {
+				"type": "object"
+			}
+		},
+		"variables": {
+			"DBforMySQLapiVersion": "2017-04-30-preview"
+		},
+		"resources": [
+			{
+				"apiVersion": "2017-04-30-preview",
+				"type": "Microsoft.DBforMySQL/servers/databases",
+				"name": "[concat(parameters('serverName'), '/', parameters('databaseName'))]",
+				"properties": {}
+			}
+		],
+		"outputs": {
+		}
+	}
+	`)
```

Comment:
In the sqldb module, we separated these ARM templates into three files. Can we do the same here, just for consistency?

Reply:
yeah, can do.